jaws-journey-deploy/orb.yml

```diff
@@ -403,6 +403,44 @@ jobs:
             - << parameters.lib >>/*
             - ./reports/jacoco/*
 
+  publish-image:
+    parameters:
+      <<: *serviceName
+    executor: kotlin
+    working_directory: ~/working
+    steps:
+      - attach_workspace:
+          at: ~/working
+      - restore_cache:
+          key: gradle-{{ checksum "build.gradle" }}-{{ checksum "<< parameters.serviceName >>/build.gradle" }}
+      - github-repo-login:
+          username: << parameters.username >>
+          token: << parameters.token >>
+      - aws-ecr/build-and-push-image:
+          setup-remote-docker: true
+          checkout: false
+          account-url: AWS_ACCOUNT_URL
+          aws-access-key-id: AWS_ACCESS_KEY_ID
+          aws-secret-access-key: AWS_SECRET_ACCESS_KEY
+          dockerfile: ./<< parameters.serviceName >>/Dockerfile
+          path: ./<< parameters.serviceName >>
+          create-repo: false
+          region: AWS_DEFAULT_REGION
+          repo: << parameters.serviceName >>
+          tag: "latest,$CIRCLE_SHA1"
+      - snyk/scan:
+          docker-image-name: $AWS_ACCOUNT_URL/<< parameters.serviceName >>:latest
```

Comment:
Could there be a race condition here if multiple jobs are running at once? Could we use `$CIRCLE_SHA1` instead?